package.json

```diff
@@ -41,6 +41,7 @@
         "icons": "wireit",
         "icons:ui": "wireit",
         "icons:workflow": "wireit",
+        "lint-staged": "lint-staged",
```

Comment:
I'm not sure we benefit from this abstraction, can we look at just using the migration work I already put together? 

https://jira.corp.adobe.com/browse/SWC-855

https://github.com/adobe/spectrum-web-components/pull/5393